webrender/src/border.rs

```diff
@@ -528,10 +528,63 @@ pub struct BorderSegmentInfo {
     widths: DeviceSize,
 }
 
+bitflags! {
+    /// Whether we depend in the available size for the border (effectively in
```

Comment:
did you mean "depend on" instead of "depend in"?

Reply:
Yup, will fix.